test/fs.js

```diff
@@ -0,0 +1,186 @@
+'use strict';
+
+const metatests = require('metatests');
+const metasync = require('metasync');
+const fs = require('fs');
+const path = require('path');
+const stream = require('stream');
+const { copyDir } = require('..');
+
+const testDataFile =
+  'JySAF1figbONWxAeeoGhDXeI7df25GKXKPcyCl63ZjIv5HZGQH8yfFeHGGaSJ9zDMP0CwtgX2madIQkFmuq38SNhcKcGQ';
+const srcDir = 'sourceDirectory';
+const targetDir = 'targetDirectory';
+const absoluteLink = __dirname.replace('test', 'lib');
+const files = [
+  '1.js',
+  '2.txt',
+  '3.html',
+  '4.css',
+  '5.doc',
+  '6.tmp',
+  '7.asm',
+  '8.zip',
+  'link',
+  'dir1',
+  'dir2',
+  'dir3',
```

Comment:
Why did you reduce the number of nested directories?

Reply:
I can change it back, my OS had limit for opened files at once.

Reply:
Well, it should be fine though I wonder how tests passed before that?